abovevtt.css

```diff
@@ -52,6 +68,9 @@ body > button:hover,
     line-height: 19px;
 }
 
+button#scene_selector_toggle {
```

Comment:
The type `button` doesn't increase the specificity of the selector. 
It might be redundant in this case.

Reply:
It is, it was copied from autogenerated selector by Chrome DevTools.
Not a blocker tho', so I might change it on a future PR.